### PR TITLE
update changelog and readme to include new partial release v0.2.0

### PR DESCRIPTION
**Pull request by Dens49**

### CODE REVIEW

**Comment by Dens49** [comment]
converted to WIP, because the v0.2.0 release isn't ready yet

**Review by katia-qm — DISMISSED** [review]
Marvellous!

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Minimum python version: **3.11**
 To install the hari-client package, use pip with the following command:
 
 ```bash
-python -m pip install "hari_client @ git+https://github.com/quality-match/hari-client@v0.1.0"
+python -m pip install "hari_client @ git+https://github.com/quality-match/hari-client@v0.2.0"
```
Comment (katia-qm) [inline README.md:12]: I think update to setup.py is missing

**Review by katia-qm — APPROVED** [review]
Marvellous2!

**Pull request merged**